### ISSUE TITLE
disabling too-many-arguments in-line when parent class is specified still produces warning

### ISSUE THREAD

**Issue by plannigan** [PyCQA/pylint#1191]
If the parent class is explicitly specified, when using an inline comment to disable `too-many-arguments`, the issue is still reported. It does not matter if the parent class is `object` or a class defined in the file. `too-many-arguments` is reported as being disabled, but the warning is still displayed.

### Steps to reproduce
```python
# pylint: disable=missing-docstring,too-few-public-methods

class NoParentClass:  # pylint: disable=too-many-arguments
    def __init__(self, arg_1, arg_2, arg_3, arg_4, arg_5, arg_6):
        self.arg_1 = arg_1
        self.arg_2 = arg_2
        self.arg_3 = arg_3
        self.arg_4 = arg_4
        self.arg_5 = arg_5
        self.arg_6 = arg_6


class ChildOfObject(object):  # pylint: disable=too-many-arguments
    def __init__(self, arg_1, arg_2, arg_3, arg_4, arg_5, arg_6):
        self.arg_1 = arg_1
        self.arg_2 = arg_2
        self.arg_3 = arg_3
        self.arg_4 = arg_4
        self.arg_5 = arg_5
        self.arg_6 = arg_6


class Base:
    pass


class ChildOfBass(Base):  # pylint: disable=too-many-arguments
    def __init__(self, arg_1, arg_2, arg_3, arg_4, arg_5, arg_6):
        self.arg_1 = arg_1
        self.arg_2 = arg_2
        self.arg_3 = arg_3
        self.arg_4 = arg_4
        self.arg_5 = arg_5
        self.arg_6 = arg_6
```

### Current behavior
```
No config file found, using default configuration
************* Module too_many_arguments
I:  1, 0: Locally disabling missing-docstring (C0111) (locally-disabled)
I:  1, 0: Locally disabling too-few-public-methods (R0903) (locally-disabled)
I:  3, 0: Locally disabling too-many-arguments (R0913) (locally-disabled)
I: 13, 0: Locally disabling too-many-arguments (R0913) (locally-disabled)
I: 27, 0: Locally disabling too-many-arguments (R0913) (locally-disabled)
R: 14, 4: Too many arguments (7/5) (too-many-arguments)
R: 28, 4: Too many arguments (7/5) (too-many-arguments)
```

### Expected behavior
`too-many-arguments` is disabled for the scope of the class in all cases and no warnings are displayed.

### pylint --version output
```
pylint --version
No config file found, using default configuration
pylint 1.6.4, 
astroid 1.4.8
Python 3.5.1 (default, Feb  7 2016, 15:03:27) 
[GCC 4.8.4]
```

**Comment by DanielNoord** [comment]
Closing as won't fix. The disable should be on the line that defines the function signature:
```python
class NoParentClass:
    def __init__(  # pylint: disable=too-many-arguments
        self, arg_1, arg_2, arg_3, arg_4, arg_5, arg_6
    ):
        self.arg_1 = arg_1
        self.arg_2 = arg_2
        self.arg_3 = arg_3
        self.arg_4 = arg_4
        self.arg_5 = arg_5
        self.arg_6 = arg_6


class ChildOfObject(object):
    def __init__(  # pylint: disable=too-many-arguments
        self, arg_1, arg_2, arg_3, arg_4, arg_5, arg_6
    ):
        self.arg_1 = arg_1
        self.arg_2 = arg_2
        self.arg_3 = arg_3
        self.arg_4 = arg_4
        self.arg_5 = arg_5
        self.arg_6 = arg_6


class Base:
    pass


class ChildOfBass(Base):
    def __init__(  # pylint: disable=too-many-arguments
        self, arg_1, arg_2, arg_3, arg_4, arg_5, arg_6
    ):
        self.arg_1 = arg_1
        self.arg_2 = arg_2
        self.arg_3 = arg_3
        self.arg_4 = arg_4
        self.arg_5 = arg_5
        self.arg_6 = arg_6
```

**Comment by DanielNoord** [comment]
I have also removed the ``issue-1191`` branch.
